R/pkg/R/DataFrame.R

```diff
@@ -1914,3 +1914,43 @@ setMethod("attach",
             }
             attach(newEnv, pos = pos, name = name, warn.conflicts = warn.conflicts)
           })
+
+#' Returns the column types of a DataFrame.
+#' 
+#' @name coltypes
+#' @title Get column types of a DataFrame
+#' @param x (DataFrame)
```

Comment:
Could you update the style of  function description to be more consistent with other existing ones?